composite_handler/engine/hooks.py

```diff
@@ -0,0 +1,2 @@
+def print_result(task):
+    print(task.result)
```

Comment:
Yeah I know it isn't required, but my intention here is to add some output of value once I have that output to add here. I've switched it now to use logging instead of printing.